pypeit/core/telluric.py

```diff
@@ -3,30 +3,33 @@
 .. include common links, assuming primary doc root is up one directory
 .. include:: ../include/links.rst
 """
+import os
+import sys
+import pickle
```

Comment:
would love to get rid of `pickle` but I'm guessing that's
not in the cards for now.

Reply:
Would a `numpy.save` work??

Reply:
Actually the module doesn't use pickle anymore, so this should be safe to remove.  I've commented it for now, but will remove it before merging.